examples/conversation.v1.js

```diff
@@ -13,24 +13,51 @@ const conversation = new watson.ConversationV1({
 });
 
 /**
- * Payload for the Watson Conversation Service
- * <workspace-id> and user input text required.
+ * Conversation service is stateless.
+ * The application need to maintain the conversation state.
+ * A context object is returned in the messgage response and the app should pass it in the next message request
  */
-const payload = {
-  workspace_id: process.env.WORKSPACE_ID || '<workspace_id>',
-  input: {
-    text: 'I am happy to talk to you today.'
-  }
+let currentContext = undefined;
```

Comment:
I think this should be a second parameter to the `message()` function, rather than a global variable. Then, below where you call `message('second message')`, you could change it to `message('second message', response1.context)`. 

I think that would make the example easier for users to build on.